url/resources/urltestdata.json

```diff
@@ -3303,12 +3303,14 @@
   {
     "input": "http:@:www.example.com",
     "base": "about:blank",
-    "failure": true
+    "failure": true,
+    "inputCanBeRelative": true
   },
   {
     "input": "http:/@:www.example.com",
     "base": "about:blank",
-    "failure": true
+    "failure": true,
+    "inputCanBeRelative": true
```

Comment:
Testing with https://jsdom.github.io/whatwg-url/#url=aHR0cDovQDp3d3cuZXhhbXBsZS5jb20=&base=aHR0cHM6Ly9leGFtcGxlLmNvbS8= implies this is correct as-is.

Reply:
new URL("http:/@:www.example.com", "http://example.com/") succeeds in Chrome, Firefox, and Safari.  I'm not sure if this is an issue with the spec or the reference implementation.

Reply:
Ah, fascinating, this only works with a base `http:` URL, not `https:`. Since WPTs run on http by default I think your change is then correct.